### PR TITLE
tizonia-openmax-il: init at 0.22.0

### PR DESCRIPTION
**Pull request by wucke13**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/tizonia-openmax-il/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  nativeBuildInputs = [ meson ninja pkg-config which ];
+
+  buildInputs = [
+    SDL bash-completion boost curl.dev dbus.dev faad2 flac.dev lame libev
+    libfishsound libmad libmediainfo liboggz libopus libpulseaudio.dev
+    libsndfile libspotify libuuid.dev libvpx libzen log4c mpg123 opusfile
+    python3 sqlite.dev taglib
+  ];
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/tizonia-openmax-il/default.nix:40]:
```suggestion
  meta = with lib; {
```

stdenv.lib got recently deprecated.

File: pkgs/development/libraries/log4c/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  src = fetchurl {
+    url = "mirror://sourceforge/log4c/${pname}-${version}.tar.gz";
+    sha256 = "1sj8xpgfmw7ncxf91a8as3avapdxynxzdysjm07w8b7mj80h54ar";
+  };
+
+  hardeningDisable = [ "format" ];
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/log4c/default.nix:15]:
```suggestion
  meta = with lib; {
```

File: pkgs/applications/audio/tizonia-openmax-il/default.nix [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    description = "Command-line streaming music client/server for Linux";
+    longDescription = ''
+      Includes support for Spotify (Premium), Google Play Music (free and paid
+      tiers), YouTube, SoundCloud, TuneIn and iHeart Internet Radio
+      directories, Plex servers and Chromecast devices.
+    '';
+    homepage = "https://tizonia.org/";
+    license = licenses.lgpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/tizonia-openmax-il/default.nix:48]: lgpl3Plus or lgpl3Only?

File: pkgs/development/libraries/log4c/default.nix [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    description = "A logging framework for C patterned after Apache log4j";
+    longDescription = ''
+      Log4c is a library of C for flexible logging to files, syslog and other
+      destinations. It is modeled after the Log for Java library .
+    '';
+    homepage = "http://log4c.sourceforge.net/";
+    license = licenses.lgpl21;
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/log4c/default.nix:22]: lgpl2Plus or lgpl2Only?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/tizonia-openmax-il/default.nix (hunk 1 above)
Comment (wucke13) [reply pkgs/applications/audio/tizonia-openmax-il/default.nix:40]: Good to know!

File: pkgs/applications/audio/tizonia-openmax-il/default.nix (hunk 3 above)
Comment (wucke13) [reply pkgs/applications/audio/tizonia-openmax-il/default.nix:48]: I don't see the "or later" clause in this: https://docs.tizonia.org/overview/COPYING.LESSER.html

File: pkgs/development/libraries/log4c/default.nix (hunk 4 above)
Comment (wucke13) [reply pkgs/development/libraries/log4c/default.nix:22]: Good catch, it is lgpl2Plus (https://sourceforge.net/p/log4c/log4c/ci/master/tree/COPYING)

File: pkgs/applications/audio/tizonia-openmax-il/default.nix [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  src = fetchFromGitHub {
+    owner = "tizonia";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "059iml5l9d7izhvci1f6piwbprzfrk12315pkifhyzbyxcd5fny9";
+  };
+
+  preConfigure = ''
+    export BOOST_INCLUDEDIR="${stdenv.lib.getDev boost}/include";
+    export BOOST_LIBRARYDIR="${stdenv.lib.getLib boost}/lib";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/tizonia-openmax-il/default.nix:21]:
```suggestion
    export BOOST_INCLUDEDIR="${lib.getDev boost}/include";
    export BOOST_LIBRARYDIR="${lib.getLib boost}/lib";
```

File: pkgs/applications/audio/tizonia-openmax-il/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/audio/tizonia-openmax-il/default.nix:40]: ofborg does not allow any additions of stdenv.lib anymore.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**